<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QVBoxLayout" name="centralVerticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page1">
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>80</y>
            <width>631</width>
            <height>441</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="WeekSched_2">
           <item>
            <widget class="QFrame" name="WeekFrame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QFrame" name="WeekFrame2_2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Weekly Schedule</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignRight">
                  <widget class="QLineEdit" name="StudentSearch">
                   <property name="placeholderText">
                    <string>Enter Student ID</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Search">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="viewCurriculum">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>View Curriculum</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="Semester">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>30</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>1st Semester</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>2nd Semester</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="WeekTable_2">
                <row>
                 <property name="text">
                  <string>7:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>8:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>9:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>10:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>11:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>12:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>1:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>2:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>3:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>4:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>5:00AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>5:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>7:00 PM</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>Sunday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Monday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Tuesday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Wednesday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Thursday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Friday</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Saturday</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>241</width>
            <height>441</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="TodaySched_3">
           <item>
            <widget class="QFrame" name="TodayFrame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item alignment="Qt::AlignLeft|Qt::AlignTop">
               <widget class="QLabel" name="TodaySched_4">
                <property name="text">
                 <string>Today's Schedule</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLabel" name="labelTodayHeader_2">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QTableWidget" name="tableWidget_2">
                <row>
                 <property name="text">
                  <string>7:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>8:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>9:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>10:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>11:00 AM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>12:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>1:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>2:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>3:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>4:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>5:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>6:00 PM</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>7:00 PM</string>
                 </property>
                </row>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="ScheduleTitle">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Schedule</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page2">
         <widget class="QFrame" name="frame1stYear">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>851</width>
            <height>441</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignRight">
            <widget class="QComboBox" name="YearBox">
             <item>
              <property name="text">
               <string>1st Year</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2nd Year</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3rd Year</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4th Year</string>
              </property>
             </item>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="FirstYearHeader">
             <property name="text">
              <string>First Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="TodaySched_7">
                <property name="text">
                 <string>1st Semester</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="TodaySched_6">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>2nd Semester</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QTableWidget" name="sem2frame">
                <column>
                 <property name="text">
                  <string>Codes</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Subject Titles</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Grades</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Units</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Pre-requisite(s)</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="sem1">
                <column>
                 <property name="text">
                  <string>Codes</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Subject Title</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Grades</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Units</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Pre-requisite(s)</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="Return">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>10</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Go Back</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
